docker-compose.yml

```diff
@@ -52,6 +58,10 @@ mysql-data:
 
 mysql:
   build: ./docker/images/mysql
+  entrypoint:
+    - docker-entrypoint.sh
```

Comment:
I picked ``docker-entrypoint.sh`` because it is the ``ENTRYPOINT`` defined in the [Dockerfile](https://github.com/docker-library/mysql/blob/9fc086343ebd36af0448438622188264d1dc2e1c/5.6/Dockerfile#L59-L61), and lists ``entrypoint.sh`` as the backward compatible name.

I am disappointed in ``docker-compose pull``, but it is good to know this behavior.